esp/src/src-react/hooks/Workunit.ts

```diff
@@ -188,6 +189,7 @@ export function useWorkunitExceptions(wuid: string): [WUInfo.ECLException[], Wor
     const [count, increment] = useCounter();
 
     React.useEffect(() => {
+        if (!workunit) return;
```

Comment:
Should probable return `[[], undefined, increment]`

Reply:
Similar to the previous comment, I think the return of useWorkunitExceptions around line 200 should be unaffected. This is preventing the inner useEffect attempting to fetch exceptions if wuid is undefined.

Reply:
Yes you are correct!